src/reducers/versions.tsx

```diff
@@ -735,6 +764,12 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
       const { selectedPath, versionId } = action.payload;
 
       const version = state.versionInfo[versionId];
+
+      if (!version) {
+        _log.error(`Version missing for versionId: ${versionId}`);
```

Comment:
> I wonder if this should throw an actual error

I guess either way works but it would be a bit better to throw an error so that Sentry gets a stack trace. It might render an error boundary and make the app unusable but at least that would be less confusing than the user seeing no change at all from their action. Thrown errors are easier to test, too 😅